client/me/profile-gravatar/index.jsx

```diff
@@ -1,27 +1,28 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	debug = require( 'debug' )( 'calypso:me:sidebar-gravatar' );
+import React from 'react';
+import _debug from 'debug';
 
 /**
  * Internal dependencies
  */
-var Gravatar = require( 'components/gravatar' ),
-	eventRecorder = require( 'me/event-recorder' );
+import Gravatar from 'components/gravatar';
+import eventRecorder from 'me/event-recorder';
 
-module.exports = React.createClass( {
+const debug = _debug( 'calypso:me:sidebar-gravatar' );
 
+export default React.createClass( {
 	displayName: 'ProfileGravatar',
 
 	mixins: [ eventRecorder ],
 
-	componentDidMount: function() {
+	componentDidMount() {
 		debug( 'The ProfileGravatar component is mounted.' );
 	},
 
-	render: function() {
-		var profileURL = '//gravatar.com/' + this.props.user.username;
+	render() {
+		const profileURL = '//gravatar.com/' + this.props.user.username;
```

Comment:
you would use string templates ...

```es6
const profileURL = `//gravatar.com/${ this.props.user.username }`;
```

... but it's just my suggestion. It's up to you.